tests/member-api.js

```diff
@@ -0,0 +1,39 @@
+import { test } from 'brittle'
+import { randomBytes } from 'crypto'
+import { KeyManager } from '@mapeo/crypto'
+
+import { MapeoRPC } from '../src/rpc/index.js'
+import { MemberApi } from '../src/member-api.js'
+import { replicate } from './helpers/rpc.js'
+
+test('Invite sends expected project-related details', async (t) => {
```

Comment:
this test kind of overlaps with some of the rpc unit tests. okay for now since I just want something here until i figure out what else to test